ucm2/alcatel-idol3/HiFi.conf

```diff
@@ -1,8 +1,11 @@
 # Use case configuration for Alcatel OneTouch Idol 3.
 # The two speakers/earpieces are connected to external codec (TFA9897)
-# Headphones support requires writing a driver for AK4375
+# Headphones use another external codec (AK4375)
 
 Define {
+	MultiMedia1Port "SEC_MI2S_RX"
+	MultiMedia4Port "SEC_MI2S_RX"
```

Comment:
You probably want QUAT here or ignore this one. This is for compressed audio playback (basically let modem/audio DSP decode mp3/aac/etc). Never figured out what's the best way to use this with multiple output ports